nvidia-dnf.py

```diff
@@ -56,6 +57,13 @@ def sack(self):
         available_kernels = sack.query().available().filter(name = KERNEL_PKG_NAME)
         driver = installed_drivers[0]
 
+        if installed_modules:
+            string_modules = ' '.join([str(elem) for elem in installed_modules])
+
+        # DKMS stream enabled
+        if installed_modules and 'dkms' in string_modules:
```

Comment:
Seems you can just do this in the first `if`? Or is `string_modules` needed later?

Reply:
I want it to `return` after it has printed all of the debug information